README.md

```diff
@@ -58,6 +67,82 @@ aws_secret_access_key=...
 aws_session_token=...
 ```
 
+### Set up using LocalStack
```

Comment:
Since this section also covers running as well as setup, change this to 

### Using LocalStack